pontoon/base/views.py

```diff
@@ -72,7 +67,10 @@ def home(request):
             return redirect('pontoon.teams.team', locale=user.profile.custom_homepage)
 
     locale = utils.get_project_locale_from_request(request, project.locales) or 'en-GB'
-    path = Resource.objects.filter(project=project, translatedresources__locale__code=locale)[0].path
+    path = Resource.objects.filter(
+        project=project,
+        translatedresources__locale__code=locale
+    ).values_list('path', flat=True)[0]
```

Comment:
There's a small optimization hidden here: I added ``values_list`` because we only care about the ``path`` field here, and nothing else.